Project.toml

```diff
@@ -0,0 +1,21 @@
+name = "ChainRulesTestUtils"
+uuid = "cdddcdb0-9152-4a09-a978-84456f9df70a"
+version = "0.1.0"
+
+[deps]
+ChainRulesCore = "d360d2e6-b24c-11e9-a2a3-2a2ae2dbcce4"
+FiniteDifferences = "26cc04aa-876d-5657-8c51-4c34ba976000"
+
+[compat]
+ChainRules = "0.3"
```

Comment:
```suggestion
```

Reply:
I thought you're supposed to `compat` test only packages as well?

Reply:
Nvm I see `scalar_rule` is in core.